my-app/src/Components/App/App.js

```diff
@@ -0,0 +1,152 @@
+import '../../index.css';
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+import api from '../../utils/WeatherApi';
+import Main from '../Main/Main';
+import Header from '../Header/Header';
+import Footer from '../Footer/Footer';
+import About from '../About/About';
+import WeatherSearch from '../WeatherSearch/WeatherSearch';
+
+export default function App() {
+  const [isLoading, setIsLoading] = React.useState(false);
+  const [isWeather, setIsWeather] = React.useState(false);
+  const [isTitle, setIsTitle] = React.useState(true);
+  const [isWeatherData, setIsWeatherData] = React.useState([]);
```

Comment:
` React.useState([]);`

`isWeatherData`  If a variable starts with `is` it means that it has a `boolean` value. So, it should be only `true` or `false`.

Please, delete `is` from the varibale names if they are not boolean

and in line 78 you set an object as a value to `isWeatherData`, so by default it should be `{}`

The check list says: The initial value of state variables contains the correct data type.